workflow/controller/controller.go

```diff
@@ -108,6 +109,7 @@ func NewWorkflowController(restConfig *rest.Config, kubeclientset kubernetes.Int
 		configController:           config.NewController(namespace, configMap, kubeclientset),
 		completedPods:              make(chan string, 512),
 		gcPods:                     make(chan string, 512),
+		cache:                      NewConfigMapCache("", namespace, kubeclientset),
```

Comment:
Why are we setting the config map name to `""`?

Reply:
I set to `""` because it later on gets set to the `configMapName` from the template, which the controller doesn't know at this point. This is relevant to the discussion about whether a user should decide this name